docs/UserGuide.md

```diff
@@ -987,14 +1003,14 @@ enterses INDEX
 <div markdown="block" class="alert alert-warning">
 
 **:warning: Warning about the usage of this command:**
-You cannot enter the session again if you are already in that particular session.
+* You cannot enter the session again if you are already in that particular session.
 
 </div>
 
 <div markdown="block" class="alert alert-success">
 
-**:green_book: Examples**:
-* `enterses 1` enters the first session and allows you to use PARTICIPATE and PRESENCE commands.
+**:green_book: Example**:
+* `enterses 1` enters the first session and allows you to use PARTICIPATE and PRESENCE commands. (**Figure 5.4.5**)
```

Comment:
Should PARTICIPATE and PRESENCE be caps actually?

Reply:
Updated!